server/services/payments-service.js

```diff
@@ -3,24 +3,17 @@ const wreck = require('@hapi/wreck').defaults({
   timeout: config.restClientTimeoutMillis
 })
 
-function buildPayload (parcelRef, id, quantity) {
-  return {
-    parcelRef,
-    actions: [{
-      action: { id },
-      options: { quantity }
-    }]
-  }
-}
-
 async function calculatePayment (parcelRef, actionId, quantity) {
+  const payloadData = JSON.stringify({ quantity })
+  console.log('In updated calculatePayment')
+  console.log(payloadData)
```

Comment:
Did you mean to leave in these as logs?

Reply:
No, they are removed in a local commit, so will be pushed before review :-)